README.md

```diff
@@ -62,11 +58,15 @@ In development, you can run `yarn dev`, which will clean the project, build, and
 
 ## Hosting
 
-The bot running on Discord as @Hanbot#9541 is hosted on Heroku [here][heroku-live]. It's owned by Ari Porad ([@ariporad][]) <[ari@ariporad.com][]>. Feel free to contact him (or ping him on the Discord) if something's gone wrong or if you have questions.
+The bot running on Discord as @Hanbot#9541 is hosted on an VPS [here][hanbot-live]. It's owned by Ari Porad ([@ariporad][]) <[ari@ariporad.com][]>. Feel free to contact him (or ping him on the Discord) if something's gone wrong or if you have questions.
+
+All commits to `master` on this repo are automatically built by [Travis CI][travis-ci], which compiles the Typescript and deploys to to the VPS. This usually takes only a few minutes.
+
+The VPS is running Ubuntu 18 with [pm2][pm2], NodeJs, and NGINX as the reverse proxy.
```

Comment:
Also, very tiny nit: JS is an acronym or a file extension, so it should either `NodeJS` or `Node.js`